checker/tests/initialization/EISOPIssue363a.java

```diff
@@ -0,0 +1,92 @@
+// Test case for issue 363:
+// https://github.com/eisop/checker-framework/issues/363
+
+import org.checkerframework.checker.initialization.qual.Initialized;
+import org.checkerframework.checker.initialization.qual.NotOnlyInitialized;
+import org.checkerframework.checker.initialization.qual.UnknownInitialization;
+
+class MyException extends Exception {
+    @NotOnlyInitialized EISOPIssue363a cause;
+
+    MyException(EISOPIssue363a cause) {
+        this.cause = cause;
+    }
+
+    MyException(@Initialized EISOPIssue363a cause, String dummy) {
```

Comment:
This just makes the `@Initialized` qualifier explicit. Is that the intention?
Should there be a separate constructor that takes an `@UnderInitialization` parameter, to test all possibilities?